snapshot-labs/sx-monorepo
# fix: offchain pending proposal voting power should not use latest block

### Summary



Closes: #940
Depends on https://github.com/snapshot-labs/sx-monorepo/pull/938

This PR fix the voting power for offchain pending proposal, which were based on `latest` block, instead of the proposal own snapshot

### How to test


1. Open the network in the dev console
2. Go to an offchain pending proposal
3. Inspect the payload for `https://score.snapshot.org/`
4. It should use a number as `snapshot`, instead of `latest`
